apps/web/app/future/team/[slug]/embed/page.tsx

```diff
@@ -0,0 +1,22 @@
+import { getServerSideProps } from "@pages/team/[slug]";
```

Comment:
getServerSideProps should be moved to the other file, because `@page/team/[slug]` uses "use client" directive.